README.md

```diff
@@ -7,23 +7,20 @@ The DataStax Kubernetes Operator for Apache Cassandra&reg;. This repository repl
 
 To create a full featured cluster, the recommend approach is to use the Helm charts from k8ssandra. Check the [Getting started](https://k8ssandra.io/docs/getting-started/) documentation at (k8ssandra.io)[https://k8ssandra.io/docs].
 
-Quick start:
-```console
-# *** This is for GKE Regular Channel - k8s 1.16 -> Adjust based on your cloud or storage options
-kubectl create -f https://raw.githubusercontent.com/k8ssandra/cass-operator/v1.7.1/docs/user/cass-operator-manifests.yaml
-kubectl create -f https://raw.githubusercontent.com/k8ssandra/cass-operator/v1.7.1/operator/k8s-flavors/gke/storage.yaml
-kubectl -n cass-operator create -f https://raw.githubusercontent.com/k8ssandra/cass-operator/v1.7.1/operator/example-cassdc-yaml/cassandra-3.11.x/example-cassdc-minimal.yaml
-```
+A more custom approach is to use Kustomize as described in the following sections if you wish to use cass-operator only.
+
+### Installing the operator with Kustomize
```

Comment:
It would be good to show some examples of installing via Kustomize. At a minimize, I would should a default namespace-scoped install and a cluster-wide install.

Reply:
I haven't finished writing these (just like the examples point to a non-existant files)